### ISSUE TITLE
Why there are TWO (same) Snapdrop apps on Google Play Store by different developers?

### ISSUE THREAD

**Issue by psydex** [RobinLinus/snapdrop#337]
https://user-images.githubusercontent.com/5594251/127855386-ed02d26d-9b4b-4f73-a0a6-7132e9ee0a1e.mp4

**Comment by fm-sys** [comment]
Hi,

I'm one of the developers 👋 



Let me tell a bit about the history:
In October 2020 I started the still actively maintained Android App project  "Snapdrop for Android" on github.

See https://github.com/fm-sys/snapdrop-android

In January 2021 the "Snapdrop | Share it" version did suddenly appear in the Playstore. I tested the app, but found many bugs and was not really convinced by the app. 

The app "Snapdrop | Share it" is totally unmaintained since that point, so I asked the developer if he would be interested in a cooperation to create one "perfect" app out of both (see https://github.com/tanujnotes/Snapdrop/issues/3). But the only response I got was "No, thanks". 

Furthermore, I find it quite strange that he uploaded his app as "early access", so that reviews are not publicly visible (but this can also be just due to my somewhat disappointed attitude towards him).  

The end of the story:
I will not take out by app from Google Play, as the other app isn't maintained and I miss many features in the other Snapdrop App e.g. Custom server URL, Userdefined device names, ... But as long as he doesn't want to discontinue his app either, there are two snapdrop apps with different features in the Playstore.

**Comment by psydex** [comment]
I would advice you to report the app to avoid user-confusion since the other app is not maintained - it might just bring bad reviews to your app

**Comment by dtantono** [comment]
Besides Android, I hope there is an iOS version of this app.

**Comment by Longshoez** [comment]
> Besides Android, I hope there is an iOS version of this app.

Well this is just a thought but my man here did use Java to develop the app, we only need somebody to port it into Kotlin and then compile it for iOS, now for getting it into the store i find that a bit difficult but today we have some alternatives for sideloading (installing) apps via the AltStore thanks mr [Riley](https://github.com/rileytestut). we would only need the .ipa file for this aproach.

**Comment by dtantono** [comment]
if you mentioned iOS sideloading via jailbreak, there are lots of security risks and loss of warranty. So, I don't think this solution is viable for everyone.

**Comment by fm-sys** [comment]
> we only need somebody to port it into Kotlin and then compile it for iOS

Switching to kotlin will unfortunately not make the app cross platform compatible. There's way more work which needs to be done.

**Comment by Longshoez** [comment]
> if you mentioned iOS sideloading via jailbreak, there are lots of security risks and loss of warranty. So, I don't think this solution is viable for everyone. 

Altstore's way of installing apps doesn't require jailbreaking your phone, it's uses a quite clever method of making the phone think you are installing your own app as a developer for testing, the only downside is that you need to refresh the app every 7 days or else it just won't open.

**Comment by dtantono** [comment]
since that method is quite uncommon to average joe. I think straight away method by download it via App store similar like Android method via play store is the best

**Comment by tanujnotes** [comment]
Hi, I'm the developer of the "other" Snapdrop Android app. I developed it just out of curiosity on a weekend. I found it very useful so I decided to upload it as the first Snapdrop app on Play Store. It's just 33kb in size, has almost 100k downloads,  thousands of happy users and one very happy developer (even if a little incompetent). Needless to say, I'm not going to take it down.

Reasons why it's published as Early Access:
1. I didn't want the app to have a bad rating because Snapdrop website is not working on certain devices.
2. I didn't want to deal with public reviews and feature requests as I already have a moderately successful FOSS app called Olauncher, which takes enough of my time.

I understand that my app has some issues. I'll fix them eventually. The reason those are not a priority because barely anyone complains about them. The biggest complaint is that the app is not working at all, which happens because Snapdrop website is not working on those devices.

Bottom line, I believe it's good that people have options. And if someone is actually serious about joining forces, instead of take-it-down request, send a pull request. Thank you.
